shell/platform/common/public/flutter_messenger.h

```diff
@@ -87,6 +88,47 @@ FLUTTER_EXPORT void FlutterDesktopMessengerSetCallback(
     FlutterDesktopMessageCallback callback,
     void* user_data);
 
+// Increments the reference count for the |messenger|.
+//
+// See also: |FlutterDesktopMessengerRelease|
+FLUTTER_EXPORT FlutterDesktopMessengerRef
+FlutterDesktopMessengerAddRef(FlutterDesktopMessengerRef messenger);
```

Comment:
Note: `AddRef` and `Release` are the established nomenclature of fml::RefPtr.

Reply:
Matches COM's names, Windows folks will feel right at home 😂